### PR TITLE
redirect to /courses

### PR DESCRIPTION
**Pull request by Erin007**
The new /courses page has all of the information from /student and /educate so those pages are redundant.  This PR redirects them to /courses. 

Redirect code.org/student to studio.code.org/courses
Redirect code.org/educate to studio.code.org/courses?view=teacher
code.org/teach redirects to /educate so will end up on studio.code.org/courses?view=teacher

### CODE REVIEW

**Comment by Erin007** [comment]
@tanyaparker

**Comment by tanyaparker** [comment]
Were you trying to save the page content in case we return to these later? I think it's okay to delete and just have a simple `.redirect` file with the redirect path only.

**Comment by tanyaparker** [comment]
LGTM - I like what you did so that the redirect preserves what environment you're on.

We're going to wait to merge this in once Brendan has made the Courses page responsive. :)

File: pegasus/sites.v3/code.org/public/educate/index.haml [hunk 1]
```diff
@@ -1,35 +1 @@
...
-  - resource_boxes << { title_s: 'Partner with us', description_t: 'Bring computer science to your country, state, region or school district.', url_s: '/educate/partner' }
-  - resource_boxes << { title_s: 'Beyond Code.org', description_t: 'There are many fantastic courses and organizations helping schools adopt computer science. See our list of recommended options.', url_s: '/educate/curriculum/3rd-party' }
-  - resource_boxes << { title_s: 'Inspire a student', description_t: 'Find a volunteer guest speaker, or use these videos, posters and how-to resources to inspire students to try computer science.', url_s: '/educate/resources/inspire' }
-  - resource_boxes << { title_s: 'Join the online community', description_t: 'Connect with other computer science teachers and stay up-to-date on what’s new with Code.org.', url_s: '/educate/community' }
-  - resource_boxes << { title_s: 'Tools and videos', description_t: 'These tools and videos can be used in any CS course to support learning.', url_s: '/educate/resources/videos' }
-  - resource_boxes.each do |box|
-    = view :educate_box, item: box, color: "#a69bc1"
+- redirect CDO.studio_url('/s/courses?view=teacher')
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/educate/index.haml]: Shouldn't this just be `/courses`? (same for others)

Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/educate/index.haml]: No, /educate and /teach are the teacher's pages so they should go to `CDO.studio_url('/courses?view=teacher')` 

There is still a correction to remove the `/s` part though.

**Comment by tanyaparker** [comment]
Brendan's changes are in. You may merge this in now. :)

**Pull request merged**